tests/Application/Security/PasswordTest.php

```diff
@@ -19,13 +19,6 @@ public function testHashSuccess(): void
         $this->assertStringStartsWith('$2y$', $hash);
     }
 
-    public function testHashWithNoInput(): void
-    {
-        $this->expectException(\TypeError::class);
-
-        Password::hash(null);
-    }
```

Comment:
deleted as passing null is now allowed by Psalm and also this aspect of the SUT is simple enough that it doesn't need testing.